src/radix.py

```diff
@@ -9,8 +9,22 @@ def count_sort(x: str) -> str:
     >>> count_sort('')
     ''
     """
-    return ""
 
+    count = {}
+    for i in range(len(x)):
+        if x[i] in count:
+            count[x[i]].append(i)
+        else:
+            count[x[i]] = [i] # {'a':[2,3]}
```

Comment:
You are only supposed to *count* how many times you see each letter here, not collect lists of their indices. You do get the sequence sorted, and in the same asymptotic time, but appending to lists is slower and uses a lot more memory.